lib/node_modules/@stdlib/repl/lib/defaults.js

```diff
@@ -87,7 +87,10 @@ function defaults() {
 			'autoDeletePairs': true,
 
 			// Flag indicating whether to enable the display of completion previews for auto-completion (note: default depends on whether TTY):
-			'completionPreviews': void 0
+			'completionPreviews': void 0,
+
+			// Flag indicating whether to enable the scrolling of long outputs:
+			'pageLongOutputs': true
```

Comment:
You are assuming that the REPL is TTY, but this may not be the case. I non-TTY contexts, paging is likely not desired (e.g., over HTTP, which is supported).

Reply:
You are right, will add a condition like we have with the preview completer👍

Reply:
How should the *to-be-implemented* `pager` command behave in non TTY contexts?

Reply:
In non-TTY, if a user invokes `pager`, we're trusting them that they know what they are doing.

Reply:
On further thought, a pager probably only makes sense in TTY. In non-TTY, we should not provide support.